### PR TITLE
Fix Fedora install

### PR DESCRIPTION
**Pull request by anadahz**

### CODE REVIEW

**Comment by hellais** [comment]
Could you rebase this on top of feature/install so that it merges cleanly?

**Comment by hellais** [comment]
Isn't this the same as #404?

File: scripts/install.sh [hunk 1]
```diff
@@ -159,27 +168,86 @@ install_obfs4proxy() {
...
       fi
       (
       set -x
-      $sh_c "yum -y install gmp-devel"
+      $sh_c "${yum} -y install gmp-devel"
+=======
+      (
+      set -x
+      $sh_c "yum -y install golang"
+>>>>>>> ed26fa9674ed2898afa35110f515c9fb8e0f6381
```
Comment (hellais) [inline scripts/install.sh]: Looks like some merge conflict went bad.

Comment (anadahz) [inline scripts/install.sh]: Indeed my bad I think it's safe to close this PR.

**Pull request closed without merging**